dfx.json

```diff
@@ -19,8 +19,14 @@
     },
     "internet_identity": {
       "type": "custom",
-      "wasm": "https://github.com/dfinity/internet-identity/releases/download/release-2024-04-05/internet_identity_dev.wasm.gz",
-      "candid": "https://github.com/dfinity/internet-identity/releases/download/release-2024-04-05/internet_identity.did"
+      "wasm": "https://github.com/dfinity/internet-identity/releases/download/release-2024-06-07/internet_identity_dev.wasm.gz",
+      "candid": "https://github.com/dfinity/internet-identity/releases/download/release-2024-06-07/internet_identity.did"
+    }
```

Comment:
I would also suggest adding 
```
"remote": {
  "id": {
    "ic": "rdmx6-jaaaa-aaaaa-aaadq-cai"
  }
}
```

This will let `dfx` know that the canister already exists no mainnet and won't deploy it there.